recipes/packio/all/test_package/conanfile.py

```diff
@@ -8,10 +8,11 @@ class TestPackageConan(ConanFile):
     generators = "cmake"
 
     def build(self):
+        packio_version = self.requires['packio'].ref.version
```

Comment:
This usage is not documented, use ''deps_cpp_info`` information to get the version https://docs.conan.io/en/latest/reference/conanfile/attributes.html#deps-cpp-info
```suggestion
        packio_version = self.deps_cpp_info["packio"].version
```